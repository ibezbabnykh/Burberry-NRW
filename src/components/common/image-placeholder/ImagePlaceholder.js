import React from 'react';

import './ImagePlaceholder.scss';

const ImagePlaceholder = () => {
  return (
    <div className="image-placeholder">
      Image not found
    </div>
  );
};

export default ImagePlaceholder;