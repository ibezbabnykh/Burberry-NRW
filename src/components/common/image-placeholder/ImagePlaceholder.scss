@import 'assets/style/variables.scss';

.image-placeholder {
    background-color: $color-message;
    display: flex;
    align-items: center;
    justify-content: center;
    width: 250px;
    height: 250px;
}