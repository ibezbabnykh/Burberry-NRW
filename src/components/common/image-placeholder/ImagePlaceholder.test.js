import ImagePlaceholder from './ImagePlaceholder';

describe('ImagePlaceholder component', () => {
  it('should render', () => {
    const component = toJson(shallow(<ImagePlaceholder />));
    expect(component).toMatchSnapshot();
  });
});