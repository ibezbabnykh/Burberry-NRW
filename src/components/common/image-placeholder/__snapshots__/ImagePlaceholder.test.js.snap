// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagePlaceholder component should render 1`] = `
<div
  className="image-placeholder"
>
  Image not found
</div>
`;
